databricks_cli/dbfs/api.py

```diff
@@ -99,7 +99,8 @@ def put_file(self, src_path, dbfs_path, overwrite):
                 contents = local_file.read(BUFFER_SIZE_BYTES)
                 if len(contents) == 0:
                     break
-                self.client.add_block(handle, b64encode(contents))
+                # add_block should not take a bytes object.
+                self.client.add_block(handle, b64encode(contents).decode())
```

Comment:
Is it just for py3 or a bug fix? If this is a correctness issue independent of py3 compatibility, we should fix it separately and make a bug fix release.

Reply:
It's just for py3.